registry/client/transport/http_reader.go

```diff
@@ -1,12 +1,18 @@
 package transport
 
 import (
-	"bufio"
 	"errors"
 	"fmt"
 	"io"
 	"net/http"
 	"os"
+	"regexp"
+	"strconv"
+)
+
+var (
+	contentRangeRegexp       = regexp.MustCompile(`bytes ([0-9]+)-([0-9]+)/([0-9]+|\\*)`)
```

Comment:
Double check the use of `=` in the byte range headers. In one case, they use and the other they don't. I am not sure if we are consistent with our non-standard use for upload (not sure if that matters here).

May just want to double check, though.